server/database_operations.js

```diff
@@ -0,0 +1,25 @@
+const { Pool } = require("db");
```

Comment:
And that should be

```js
import { Pool } from "pg";
```

As Pool is a Postgres module

Reply:
ah, that works too. I thought we would just import db from "./db"